.github/workflows/ci_passkey.yml

```diff
@@ -0,0 +1,32 @@
+name: safe-modules-passkey
+on: [push]
```

Comment:
I'd use the paths filter to run it only when this actual module is changed: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
Added paths filters to CI jobs.